test/system/075-exec.bats

```diff
@@ -230,6 +230,17 @@ load helpers
     assert "$output" = "0" ".ExecIDs must be empty"
 }
 
+@test "podman exec error codes" {
+    run_podman run -d $IMAGE top
+    cid="$output"
+    run_podman 126 exec $cid /etc
```

Comment:
Already tested way way way at top, line 29, and with special-case code for runc.